### PR TITLE
[SCAL-215364] Add config to enable data panel v2 when only liveboard is embedded

### PR DESCRIPTION
**Pull request by bishalts2019**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -1040,6 +1040,19 @@ export interface ViewConfig {
      * @version SDK: 1.32.1 | ThoughtSpot: 10.3.0.cl
      */
     disableRedirectionLinksInNewTab?: boolean;
+    /**
+     * Flag to control Data panel experience
+     * @default true
+     * @version SDK: 1.26.0 | Thoughtspot: 9.7.0.cl
```
Comment (utsavkapoor) [inline src/types.ts:1046]: Fix the SDK Version and Thoughtspot version

File: src/embed/liveboard.spec.ts [hunk 2]
```diff
@@ -57,12 +57,13 @@ describe('Liveboard/viz embed tests', () => {
...
             ...defaultViewConfig,
             liveboardId,
+            dataPanelV2: true,
         } as LiveboardViewConfig);
         liveboardEmbed.render();
         await executeAfterWait(() => {
             expectUrlMatchesWithParams(
                 getIFrameSrc(),
-                `http://${thoughtSpotHost}/?embedApp=true${defaultParams}${prefixParams}#/embed/viz/${liveboardId}`,
+                `http://${thoughtSpotHost}/?embedApp=true${defaultParams}${prefixParams}&enableDataPanelV2=true#/embed/viz/${liveboardId}`,
```
Comment (utsavkapoor) [inline src/embed/liveboard.spec.ts:66]: Lets not do this. Can you add a new UT for this ?

**Review by utsavkapoor — CHANGES_REQUESTED** [review]

File: src/embed/liveboard.spec.ts (hunk 2 above)
Comment (bishalts2019) [reply src/embed/liveboard.spec.ts:66]: sure

File: src/types.ts (hunk 1 above)
Comment (bishalts2019) [reply src/types.ts:1046]: done

**Review by utsavkapoor — APPROVED** [review]

**Pull request merged**